Makefile

```diff
@@ -107,6 +107,9 @@ build-kind-images: build-kind-images-ko
 install: WHAT ?= ./cmd/...
 install:
 	go install -ldflags="$(LDFLAGS)" $(WHAT)
+	$(eval KUBECTL_WORKSPACES=$(shell which kubectl-workspaces))
```

Comment:
Don't have to do it in this PR, but since the `Use` field is `workspace` and the package is `pkg/cliplugins/workspace` (both singular), should we consider renaming `cmd/kubectl-workspaces` to `cmd/kubectl-workspace` and then having `kubectl-workspaces` be a symlink to `kubectl-workspaces`?

Reply:
This assumes that `${GOPATH}/bin` is in the user's `${PATH}` which may not be true (is not true for me, for instance). We could look at `${GOBIN}` and the defaults explicitly, perhaps:

> Executables are installed in the directory named by the GOBIN environment
variable, which defaults to $GOPATH/bin or $HOME/go/bin if the GOPATH
environment variable is not set. Executables in $GOROOT
are installed in $GOROOT/bin or $GOTOOLDIR instead of $GOBIN.

Reply:
> This assumes that ${GOPATH}/bin is in the user's ${PATH} which may not be true 

Ack, I made this assumption because the [kubectl plugin docs](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/) say plugins are discovered via `$PATH`

I did also consider parsing the `go env` output, but as you say that is complicated by the fact we have to detect whether to use `$GOBIN` or `$GOPATH/bin` (GOBIN isn't actually defaulted in the `go env` output even if it is internally for `go install` ...), so after looking at the plugin docs I decided to use `which` instead.

I'll see if there's a clean way to handle this case, thanks for the feedback!

Reply:
+1 for GOBIN. Build path does not have to equal the users' kubectl plugin in use and hence in the PATH.

Reply:
Sorry for the slow update, this now handles GOBIN and I moved the cmd directory to workspace (singular) as requested by @ncdc - thanks for the review!